test/check-verify

```diff
@@ -94,8 +94,6 @@ def install(opts):
         cmd = [ "./testsuite-prepare" ]
         if opts.clean:
             cmd.append("--clean")
-        if opts.verbosity:
-            cmd.append("--verbose")
```

Comment:
This can be debated. `check-verify` is verbose by default, but `testsuite-prepare --verbose` is now a lot more verbose than before, so it might be distracting.